armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -39,16 +39,11 @@
     _determineLinked,
 )
 from armi import materials
-from armi.materials import custom
+from armi.materials import custom, _MATERIAL_NAMESPACE_ORDER
 from armi.reactor.flags import Flags
 from armi.tests import mockRunLogs
 from armi.utils import units
 
-# set namespace order for materials so that fake HT9 material can be found
-materials.setMaterialNamespaceOrder(
-    ["armi.reactor.converters.tests.test_axialExpansionChanger", "armi.materials"]
-)
-
 
 class Base(unittest.TestCase):
```

Comment:
In retrospect, I think `Base` is too generic a name.  Tests are importable from ARMI, so this means the ARMI API has one "Base" class people could use, and it's this one. I expect that would surprise the users.

Can we rename this `AxialExpansionTestBase` (or something similar)?

Reply:
Great idea. I like it. 
https://github.com/terrapower/armi/pull/1046/commits/f5226c475ba0ebf6856c0a5d7556859a58b9fd47